inf-clojure.el

```diff
@@ -702,6 +746,35 @@ is present it will be used instead of the current file."
     (when switch-to-repl
       (inf-clojure-switch-to-repl t))))
 
+(defcustom inf-clojure-reload-default t
+  "Controls whether `inf-clojure-reload' should reload or reload-all by default."
+  :type '(choice (const :tag "Reload" t)
+                 (const :tag "Reload All" nil))
+  :safe #'booleanp
+  :package-version '(inf-clojure . "2.2.0"))
+
+(defun inf-clojure-reload (arg)
+  "Send a query to the inferior Clojure for reloading the namespace.
+See variable `inf-clojure-reload-form' and
+`inf-clojure-reload-all-form'.
+
+The prefix argument ARG can change the behavior of the command:
+
+  - C-u M-x `inf-clojure-reload': prompts for a namespace name.
+  - M-- M-x `inf-clojure-reload': inverts the default reload command
+  - M-- C-u M-x `inf-clojure-reload': inverts default AND prompts for a namespace name."
+  (interactive "P")
+  (let* ((proc (inf-clojure-proc))
+         (invert? (or (equal current-prefix-arg "-") (equal current-prefix-arg '(-4))))
+         (prompt? (or (equal current-prefix-arg '(4)) (equal current-prefix-arg '(-4))))
```

Comment:
Remember we don't use `?` in Emacs - it's `p` there.

Reply:
Right!